src/config/dynamic-theme-fixes.config

```diff
@@ -12772,6 +12772,7 @@ a[href*="about/products"][title]
 .JOmIqc
 .hLcKi
 #EcMbV
+#xRRFWe
```

Comment:
I think a better fix is something along the lines:

```suggestion
#wrap a[href="/mobile/"]
```

So it doesn't rely on the obscure selector

Reply:
Reference:

![image](https://user-images.githubusercontent.com/25481501/119264246-dcafcd80-bbd1-11eb-9ab4-3ae5616d3e20.png)

Reply:
Sounds good to me!